### ISSUE TITLE
News flyout hover text color

### ISSUE THREAD

**Issue by shieldsb** [osulp/main-site#147]
When you hover over a news item in the flyout the text color changes to black. Sagan noted the accessibility issues with this. This could be happening with events too, but there aren't any items appearing now to test. 

![Image](https://github.com/user-attachments/assets/3675e6b0-2f2c-4b7a-b81d-f6cd71b80dbc)
